pkg/detectors/ipinfo/ipinfo.go

```diff
@@ -0,0 +1,89 @@
+package ipinfo
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(`\b([a-f0-9]{14})\b`)
```

Comment:
@lc this is too generic and will match a lot of false positives. `detectors.PrefixRegex([]string{"ipinfo"})` would tighten it up a bit

Reply:
Hey @zricethezav,

Done, thanks